go/proxyd/server.go

```diff
@@ -38,8 +45,10 @@ func NewServer(
 func (s *Server) ListenAndServe(host string, port int) error {
 	hdlr := mux.NewRouter()
 	hdlr.HandleFunc("/healthz", s.HandleHealthz).Methods("GET")
-	hdlr.HandleFunc("/", s.HandleRPC).Methods("POST")
-	hdlr.HandleFunc("/ws", s.HandleWS)
+	hdlr.HandleFunc("/api/v1/rpc", s.HandleRPC).Methods("POST")
+	hdlr.HandleFunc("/api/v1/{authorization}/rpc", s.HandleRPC).Methods("POST")
+	hdlr.HandleFunc("/api/v1/ws", s.HandleWS)
```

Comment:
These need the `.Methods("POST")` added to the end

Reply:
WebSockets are not technically POST requests, so we need to leave these unannotated. Adding `.Methods("POST")` to these endpoints returns a `405 Method Not Allowed`.